[docs] add Strivacity authentication example in the OAuth providers guide (#30690)
expo/expo: docs/pages/guides/authentication.mdx

</Box>
{/* End Strava */}

<Box
  name="Strivacity"
  createUrl="https://www.strivacity.com/learn-support/developer-hub"
  image={ASSETS.strivacity}
>

> | Website                     | Provider  | PKCE      | Auto Discovery |
> | --------------------------- | --------- | --------- | -------------- |
> | [Get Your Config](https://docs.strivacity.com/docs/setting-up-your-instance) | OpenID | Supported | Available  |

Reviewer: ```suggestion
| Website                                                                      | Provider | PKCE      | Auto Discovery |
| ---------------------------------------------------------------------------- | -------- | --------- | -------------- |
| [Get Your Config](https://docs.strivacity.com/docs/setting-up-your-instance) | OpenID   | Supported | Available      |
```

</Box>
{/* End Strava */}

<Box
  name="Strivacity"
  createUrl="https://www.strivacity.com/learn-support/developer-hub"
  image={ASSETS.strivacity}
>

| Website                     | Provider  | PKCE      | Auto Discovery |
| --------------------------- | --------- | --------- | -------------- |

> | [Get Your Config][c-strivacity] | OpenID | Supported | Available  |

Reviewer: ```suggestion
| [Get Your Config](https://docs.strivacity.com/docs/setting-up-your-instance) | OpenID | Supported | Available  |
```

> 
> [c-strivacity]: https://docs.strivacity.com/docs/setting-up-your-instance

Reviewer: ```suggestion
```

Since there's only one time this linked is used, we don't neet to define it like this.

{/* prettier-ignore */}
```tsx Strivacity Auth Example
import * as React from 'react';
import * as WebBrowser from 'expo-web-browser';
import { makeRedirectUri, useAuthRequest } from 'expo-auth-session';
import { Button } from 'react-native';

> /* @info <strong>Web only:</strong> This method should be invoked on the page that the auth popup gets redirected to on web, it'll ensure that authentication is completed properly. On native this does nothing. */

Reviewer: ```suggestion
/* @info <strong>Web only:</strong> This method should be invoked on the page that the auth popup gets redirected to on web. It will ensure that authentication is completed properly. On native platforms, this does nothing. */
```
